.ci/scripts/send-pr.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -uexo pipefail
+
+readonly APM_AGENT=${1}
+
+echo "Sending PR to apm-$APM_AGENT-agent"
```

Comment:
I'm going to elaborate a bit more the shell script with hub to send the PR in follow-up commits

Reply:
Added the script